bcda/web/router.go

```diff
@@ -34,6 +35,14 @@ func NewAPIRouter() http.Handler {
 		r.With(auth.RequireTokenAuth, auth.RequireTokenJobMatch).Get(m.WrapHandler("/jobs/{jobID}", jobStatus))
 		r.Get(m.WrapHandler("/metadata", metadata))
 	})
+
+	if utils.GetEnvBool("VERSION_2_ENDPOINT_ACTIVE", false) {
+		r.Route("/api/v2", func(r chi.Router) {
+			r.With(auth.RequireTokenAuth, ValidateBulkRequestHeaders).Get(m.WrapHandler("/Patient/$export", v2.BulkPatientRequest))
+			r.With(auth.RequireTokenAuth, ValidateBulkRequestHeaders).Get(m.WrapHandler("/Group/{groupId}/$export", v2.BulkGroupRequest))
```

Comment:
Job status endpoint?

Reply:
The ticket says only to move over the patient and group endpoints